feature_tracker/CMakeLists.txt

```diff
@@ -1,37 +1,62 @@
-cmake_minimum_required(VERSION 2.8.3)
-project(feature_tracker)
-
-set(CMAKE_BUILD_TYPE "Release")
-set(CMAKE_CXX_FLAGS "-std=c++17")
-set(CMAKE_CXX_FLAGS_RELEASE "-O3 -Wall -g")
-
-find_package(catkin REQUIRED COMPONENTS
-    roscpp
-    std_msgs
-    sensor_msgs
-    cv_bridge
-    )
+cmake_minimum_required(VERSION 3.0)
+project(feature_tracker VERSION 1.0)
+set(CMAKE_CXX_FLAGS "-std=c++17 -DEIGEN_DONT_PARALLELIZE")
```

Comment:
Unsure about the necessity of the DEIGEN_DONT_PARALLELIZE flag

Reply:
Will remove. Artefact from copying. Guilty.